### PR TITLE
Small fixes after initial read through of the whole lesson

### PR DESCRIPTION
**Pull request by anenadic**

### CODE REVIEW

File: episodes/01-introduction.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ is covered in
 
 :::::::::::::::::::::::::::::::::::::::: keypoints
 
-- This training aims to teach you a process for designing and developing a lesson website, in collaboration with others.
+- This training aims to teach you a process for designing and developing an online lesson material, in collaboration with others.
```
Comment (tobyhodges) [inline episodes/01-introduction.md:145]: I think I prefer the original wording. Can you elaborate on why you think "online lesson material" is more appropriate?

File: episodes/07-infrastructure.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ this has been how The Carpentries presents its lesson websites to the world.
 
 ## Using The Carpentries Workbench
 
-Carpentries lesson websites are built with **The Carpentries Workbench**,
+Carpentries lesson websites are built with [**The Carpentries Workbench**](https://carpentries.github.io/sandpaper-docs/),
```
Comment (tobyhodges) [inline episodes/07-infrastructure.md:51]:
```suggestion
Carpentries lesson websites are built with [**The Carpentries Workbench**](https://carpentries.github.io/workbench/),
```

I think this is the preferred target for Workbench links now.

File: episodes/11-exercises.md [hunk 3]
```diff
@@ -123,7 +121,7 @@ LOs should be tailored to the audience, and,
 if this is well done, you may stop worrying about the audience.
 Create LOs for the specific audience and create assessments for specific LOs."*
 
-\- Dr. Allegra Via, Carpentries Instructor Trainer
+\- Dr. Allegra Via, The Carpentries Instructor Trainer
```
Comment (tobyhodges) [inline episodes/11-exercises.md:124]: "Carpentries Instructor Trainer" (without "The") is fine according to [the style guide](https://docs.carpentries.org/topic_folders/communications/resources/style-guide.html#carpentries), and helps avoid ambiguity about whether Allegra is our only Instructor Trainer or not! 😅

File: episodes/15-explanation.md [hunk 4]
```diff
@@ -207,7 +207,7 @@ data instead.
 
 When writing a lesson, you should also consider adding key terms to the lesson glossary 
 for the lesson.
-Following the instructions on [how to create a glossary in the workbench documetation][sandpaper-docs-learners]
+Following the instructions on [how to create a glossary in the Workbench documetation][sandpaper-docs-learners]
```
Comment (tobyhodges) [inline episodes/15-explanation.md:210]:
```suggestion
Following the instructions on [how to create a glossary in the Workbench documentation][sandpaper-docs-learners]
```

File: index.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ Before joining Collaborative Lesson Development training, participants should be
 
 After attending this training, participants will be able to:
 
-- collaboratively develop and publish lessons using The Carpentries lesson infrastructure:
+- collaboratively develop and publish lessons using The Carpentries lesson infrastructure (aka [The Carpentries Workbench](https://github.com/carpentries/sandpaper-docs)):
```
Comment (tobyhodges) [inline index.md:25]:
```suggestion
- collaboratively develop and publish lessons using The Carpentries lesson infrastructure (aka [The Carpentries Workbench](https://carpentries.github.io/workbench/)):
```

@anenadic please could you add this URL as a `[workbench]` entry to the `links.md` file and remake these links "reference style" using that target?

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Thank you for these improvements @anenadic 🙌 I have suggested a more appropriate URL for the Workbench links, and am pushing back gently against a couple of your changes 😄

File: episodes/01-introduction.md (hunk 1 above)
Comment (anenadic) [reply episodes/01-introduction.md:145]: I suppose we are teaching them both - the lesson design (nothing to do with the website) as well as using the Workbench to implement a lesson as a lesson website. 

Happy to receive pushbacks on comments ;-). And happy for this to stay as originally written - will go with your call.

File: episodes/07-infrastructure.md (hunk 2 above)
Comment (anenadic) [reply episodes/07-infrastructure.md]: Great - I was not away and could not google it out for some reason.

File: episodes/11-exercises.md (hunk 3 above)
Comment (anenadic) [reply episodes/11-exercises.md:124]: 👍

File: index.md (hunk 5 above)
Comment (anenadic) [reply index.md]: Will do.

File: episodes/01-introduction.md (hunk 1 above)
Comment (tobyhodges) [reply episodes/01-introduction.md:145]: ah I see your point now. how about this?

```suggestion
- This training aims to teach you a process for designing a lesson and the skills to develop it as an open source website, in collaboration with others.
```

Comment (anenadic) [reply episodes/01-introduction.md:145]: OK, will add that now 👍

**Review by tobyhodges — APPROVED** [review]

**Comment by tobyhodges** [comment]
Thank you @anenadic 😄

**Comment by anenadic** [comment]
I think this is ready to merge now @tobyhodges.

**Pull request merged**